jest.config.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
   collectCoverageFrom: ['src/**/.(ts|tsx|js|jsx)$', '!src/**/*.test.(ts|tsx|js|jsx)$', '!src/**/*.d.ts'],
   coverageDirectory: 'coverage',
   projects: [
+    // '<rootDir>/packages/amplify-category-api',
```

Comment:
why are we disabling coverage of the plugin package?

Reply:
You'll notice the change doesn't disable anything, it adds a commented out line. We have a backlog item to explore the purpose of this setting, but the most immediate effect is that adding it uncommented enables VSCode's Jest integration for that package, making in-app testing a lot easier. I commented it out after I was done testing to remind us of that package's existence when the time comes to review.